### PR TITLE
add css-minimizer-webpack-plugin for Webpack 5

### PR DESCRIPTION
**Pull request by uidoyen**
Use `css-minimizer-webpack-plugin` instead of `Optimize CSS Assets Webpack Plugin` for Webpack 5

### CODE REVIEW

File: ui/packages/ui/package.json [hunk 1]
```diff
@@ -73,6 +73,7 @@
     "@patternfly/react-core": "^4.75.2",
     "@patternfly/react-icons": "^4.7.16",
     "@patternfly/react-table": "^4.19.24",
+    "css-minimizer-webpack-plugin": "^1.2.0",
```
Comment (indraraj) [inline ui/packages/ui/package.json:76]: it should be in devDependencies

**Comment by indraraj** [comment]
remove `"optimize-css-assets-webpack-plugin": "^5.0.4",` from package.json

File: ui/packages/ui/package.json (hunk 1 above)
Comment (uidoyen) [reply ui/packages/ui/package.json]: Moved to `devDependencies`

**Comment by uidoyen** [comment]
> remove `"optimize-css-assets-webpack-plugin": "^5.0.4",` from package.json

Removed.

**Review by indraraj — APPROVED** [review]

**Pull request merged**